### PR TITLE
Add Suricata 7.0.0 IDS/IPS

### PR DESCRIPTION
**Pull request by jamonation**
Adds: [suricata](https://github.com/OISF/suricata) 7.0.0

Related: #4062 #4063 

#### For new package PRs only
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates

### CODE REVIEW

**Comment by jamonation** [comment]
This needs #4062 and #4063 to be merged in first, since those are dependencies of Suricata proper.

**Comment by jamonation** [comment]
Something not working again with my local build, marking as a draft until I can reliably rebuild in a clean container.

File: suricata.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      install -d -m0750 ${{targets.destdir}}/var/log/suricata
+
+      make -j1 DESTDIR=${{targets.destdir}} install-full
+  
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: suricataland/suricata
```
Comment (ajayk) [inline suricata.yaml:85]:
```suggestion
    identifier: OISF/suricata
```

**Comment by rawlingsj** [comment]
I think we'll need to figure out what's happening with aarch64 else downstream projects wont be able to build consistent images.

**Comment by jamonation** [comment]
It turns out the issue with `aarch64` appears to be with building on macOS using Docker. I tried on a native Linux `aarch64` system and the build works:

```
[sdk] ❯ suricata -V
This is Suricata version 7.0.0 RELEASE

[sdk] ❯ file /usr/bin/suricata
/usr/bin/suricata: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 4.9.0, stripped
```

So I'll update the PR to remove the `target-architecture` section, and let CI do its thing to make sure this is safe to merge.

**Review by joshrwolf — APPROVED** [review]
lets make sure to monitor this as it lands and finally attempts to build on aarch64 🎉

**Comment by jamonation** [comment]
Now that CI does multi-arch, if this passes I hope it ought to be good to merge

**Pull request merged**